01_01_Basic_Rules.txt

```diff
@@ -379,10 +386,10 @@ object depending on how heavy it is.
 
 ===== Nanites =====
 
-	Although not a primary or secondary stat per se, The maximum number of 
-Nanites you can use at a given time is determined by your character class and an 
-associated primary stat. Your class may tell you its "Class Primary Attribute" 
-is Strength. In that case, Strength * 10 is your maximum Nanites. 
+	The maximum number of Nanites you can use at a given time is determined by 
+your character class and an associated Primary Stat. Your class may tell you its 
+"Class Primary Attribute" is Strength. In that case, Strength * 10 is your 
```

Comment:
This is obsolete, please scope creep to kill it?

Reply:
I thought I changed all the references to old nanites?

Reply:
I was wondering about that. Ya, I'll grab it.